docs/readme-mbed.md

```diff
@@ -0,0 +1,41 @@
+# MCUboot port for Mbed OS
+
+This is an MCUboot port for Mbed OS.
+
+## Using MCUboot
+
+Note: The following is a general overview. It does not cover MCUboot or Mbed OS basics.
+
+See https://github.com/AGlass0fMilk/mbed-mcuboot-demo as a detailed example.
+
+### Basic configurations
+
+To use MCUboot, you need to create an Mbed OS project with the following configurations:
+* `"mcuboot.primary-slot-address"`: address of the primary slot in the internal flash
+* `"mcuboot.slot-size"`: size of an image slot (only one image, two slots are currently supported)
+* `"mcuboot.max-img-sectors"`: maximum number of sectors, should be at least the number of sectors in each slot
+* `"target.restrict_size"`: the maximum size of the bootloader, such that it does not overlap with the primary slot
+
+More configurations such as signing algorithm, slot swapping, etc. can be found in [mbed_lib.json](../boot/mbed/mbed_lib.json). Please note that certain features are not currently supported.
```

Comment:
The link should be changed from `../boot/mbed/mbed_lib.json` to `https://github.com/JuulLabs-OSS/mcuboot/tree/master/boot/mbed/mbed_lib.json`